kuri_camera/CMakeLists.txt

```diff
@@ -1,53 +1,76 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(kuri_camera)
-
+# NOTE: You may have to change the following paths
+set(libav_path /usr/lib/x86_64-linux-gnu)
```

Comment:
Hmm, it wasn't able to find the libav/libsws libraries on my computer. 

Also, the find_file setup (as opposed to directly doing target_link_libraries) is nice, because the kuri_camera package has two types of nodes: those that need to be run on Kuri (e.g. uds_to_tcp) and those that need to be run on the remote computer (e.g. h264_decoder_node). Kuri does not need to compile the nodes that will never be run on Kuri. Therefore, I'd like to maintain the setup where I first check to see if certain libraries exist, and then decide which nodes to build.

Reply:
That's a good pattern! `find_library` should be able to sub in here without having to hardcode the search path.

Reply:
Done -- thanks for the suggestion!